src/pages/ThorChainLP/PoolsPage.tsx

```diff
@@ -1,17 +1,36 @@
 import type { GridProps } from '@chakra-ui/react'
+import { lazy } from 'react'
 import { Route, Switch, useRouteMatch } from 'react-router'
-
-import { AddLiquidityPage } from './AddLiquidityPage'
-import { AvailablePools } from './AvailablePools'
-import { Pool } from './Pool/Pool'
-import { Position } from './Position/Position'
-import { YourPositions } from './YourPositions'
+import { makeSuspenseful } from 'utils/makeSuspenseful'
 
 export const poolRowGrid: GridProps['gridTemplateColumns'] = {
   base: 'minmax(150px, 1fr) repeat(1, minmax(40px, max-content))',
   md: '200px repeat(5, 1fr)',
 }
 
+const AvailablePools = makeSuspenseful(
```

Comment:
let me know if we don't want to do this, but seems like avoid fetch until user interacts has been the desired outcome to avoid making initial load slower

Reply:
Actually sounds sane to me - we're already dealing with overfetching and perf issues so if this can paranoia ensure that nothing gets fetched until the user is *actually* navigating to the pools page (which should already be the case but paranoia doesn't hurt) + lazy suspense-load the components themselves, might as well!